scripts/create-metadata-from-assets.js

```diff
@@ -0,0 +1,65 @@
+const del = require('del');
+const path = require('path');
+const faker = require('faker');
+const fs = require('fs');
+const makeDir = require('make-dir');
+const Confirm = require('prompt-confirm');
+
+const { getIPFSHash } = require('../src/utils/ipfs-uri');
+
+const RESOURCES_FOLDER = path.resolve(__dirname, '..', 'resources');
+const ASSETS_FOLDER = path.join(RESOURCES_FOLDER, 'assets');
+const METADATA_FOLDER = path.join(RESOURCES_FOLDER, 'metadata');
+const IPFS_PREFIX = 'ipfs://';
+
+function fileExists(file) {
+  return fs.promises
+    .access(file, fs.constants.F_OK)
+    .then(() => true)
+    .catch(() => false);
+}
+
+async function main() {
+  if (await fileExists(path.join(METADATA_FOLDER, '0.json'))) {
+    const confirm = new Confirm('Do you want to recreate the assets metadata?');
+    
+    const yes = await confirm.run();
+
+    if (!yes) process.exit();
+
+    await del([METADATA_FOLDER]);
+  }
+
+  const assets = await fs.promises.readdir(ASSETS_FOLDER);
+
+  if (!assets.length) throw new Error('Assets are needed');
+
+  await makeDir(METADATA_FOLDER);
+
+  return Promise.all(assets.map( async (asset) => {
+    const filepath = path.join(ASSETS_FOLDER, asset);
+    const metadataPath = path.join(METADATA_FOLDER, `${path.parse(asset).name}.json`);
+
+    const file = await fs.promises.readFile(filepath);
+
+    const ipfsHash = await getIPFSHash(file);
+
+    const metadata = {
```

Comment:
Based on what we talked to @deadpine, she will create some fields of the metadata, so we will need to modify the existing files to add the ipfs hash and others fields.